packages/contracts-bedrock/test/L1/OptimismPortal.t.sol

```diff
@@ -894,19 +909,14 @@ contract OptimismPortalUpgradeable_Test is CommonTest {
         assertEq(prevBlockNum, block.number);
     }
 
-    /// @dev Tests that the proxy cannot be initialized twice.
-    function test_initialize_cannotInitProxy_reverts() external {
-        SuperchainConfig superchainConfig = SuperchainConfig(deploy.mustGetAddress("SuperchainConfig"));
-        vm.expectRevert("Initializable: contract is already initialized");
-        optimismPortal.initialize(superchainConfig);
-    }
-
```

Comment:
we can delete this because we are checking for reinitialization in `Initializable.t.sol` of both the proxy and the implementation of OptimismPortal